src/wasm-link.h

```diff
@@ -94,24 +94,22 @@ struct Section {
    * section. */
   size_t output_payload_offset;
 };
-WABT_DEFINE_VECTOR(section, Section);
-
-typedef Section* SectionPtr;
-WABT_DEFINE_VECTOR(section_ptr, SectionPtr);
-
-WABT_DEFINE_VECTOR(string_slice, StringSlice);
+typedef std::vector<Section*> SectionPtrVector;
```

Comment:
Looks like there is a newline missing above this line, although unrelated to this change.